homeassistant/components/image_processing/facebox_face_detect.py

```diff
@@ -0,0 +1,90 @@
+"""
+Component that will perform facial detection via a local facebox instance.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/image_processing.facebox_face_detect
+"""
+import base64
+import requests
```

Comment:
This is a 3rd party library and should be grouped with those ie voluptuous.

Reply:
Ok